### PR TITLE
chore: forbid wildcard imports + additional rustfmt

### PR DESCRIPTION
**Pull request by tychoish**
(this is all `cargo clippy --fix`)

### CODE REVIEW

**Review by scsmithr — COMMENTED** [review]
Needs a format. I'm good with the change though.

**Review by scsmithr — APPROVED** [review]

File: xtask/Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,8 @@ name = "xtask"
 version = "0.1.0"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lints]
+workspace = true
```
Comment (universalmind303) [inline xtask/Cargo.toml:7]: why is this needed? isnt this the default behavior?
